sdk/ml/azure-ai-ml/azure/ai/ml/entities/_assets/asset.py

```diff
@@ -14,20 +14,24 @@
 
 
 class Asset(Resource):
-    """Base class for asset, can't be instantiated directly.
-
-    :param name: Name of the resource.
-    :type name: str
-    :param version: Version of the asset.
-    :type version: str
-    :param description: Description of the resource.
-    :type description: str
-    :param tags: Tag dictionary. Tags can be added, removed, and updated.
-    :type tags: dict[str, str]
-    :param properties: The asset property dictionary.
-    :type properties: dict[str, str]
+    """Base class for asset.
+
+    This class should not be instantiated directly. Instead, use one of the child classes.
+
+    :param name: The name of the asset. Defaults to a random GUID.
+    :type name: Optional[str]]
+    :param version: The version of the asset. Defaults to "1" if no name is provided, otherwise defaults to autoincrement
+        from the last registered version of the asset with that name. For a model name that has never been registered,
+        a default version will be assigned.
```

Comment:
Do we know what the default version is? Is it "1" like in model.py?

Reply:
It differs across the asset types - that's why a specific format isn't specified here in the base class